build/linux/installer/conf/fluent-bit-common.conf

```diff
@@ -84,3 +84,11 @@
     Retry_Limit                     10
     TelemetryPushIntervalSeconds    300
     Match                           oms.container.*
+
+[OUTPUT]
```

Comment:
Did you verify running multiple instances of oms output plugin doesnt cause any concurrency issues?

Reply:
yes i did not see any